js/config/styles 2.js

```diff
@@ -0,0 +1,48 @@
+export const colors = {
```

Comment:
styles 2.js is not a good name. Please rename this file.

Reply:
no idea where the files 2 came from. I have deleted them on that branch and pushed back up.